<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ICInstructModifyDialog</class>
 <widget class="QDialog" name="ICInstructModifyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/resource/parameter.png</normaloff>:/resource/parameter.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="positionLabel">
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="posEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="mmLabel">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="speedLabel">
     <property name="text">
      <string>Speed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="speedEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="precentLabel">
     <property name="text">
      <string>%</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="positionLabel_3">
     <property name="text">
      <string>Delay Time</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="delayTimeEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="positionLabel_6">
     <property name="text">
      <string>s</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="earlyEndCheckBox">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="text">
      <string>Early End</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="earlyEndTimeEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="mmLabel_2">
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="setButton">
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="selectLabel">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="selectEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QCheckBox" name="badProductBox">
     <property name="styleSheet">
      <string notr="true">::indicator {
     width: 32px;
     height: 32px;
 }

::indicator:unchecked 
{
     image: url(:/resource/check_unchecked.png);
 }
::indicator:checked 
{
     image: url(:/resource/check_checked.png);
 }

QRadioButton::indicator:checked:disabled
{
 image: url(:/resource/check_checked_disable.png);
}

QRadioButton::indicator:unchecked:disabled
{
 image: url(:/resource/check_unchecked_disable.png);
}



</string>
     </property>
     <property name="text">
      <string>Bad Product</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ICInstructModifyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ICInstructModifyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
